### DISCUSSION TITLE
Looking for Assistance Please

### DISCUSSION THREAD

**Discussion by TrigMan59** [community/community#61195] [Programming Help]
Hi Folks,

I am a complete newbie when it comes to programming and even with AI assistance, I'm not sure I can write code. I'm in need of a simple (at least I think it should be) program to help me. I have Parkinson's Disease and watch a lot of helpful Zoom webinars that are related to my illness. Some of them are recorded and then put on YouTube, but some are not. I really need a program running on my Windows 10 PC that can do the following:

1.Start program
2. Open a dialog box to input the start time, webinar length and meeting URL to execute the program
3.At the set time open Firefox and go to input URL
4. Open Zoom
5. Turn off video
6. Mute microphone
7. Speaker view, Full screen
8. Start OBS Studio
9. Start recording
10. After the program length time runs out, stop recording
11. Stop OBS Studio
12, Stop Zoom
13. End program

As someone on a fixed income due to my disability, unfortunately I cannot afford to hire a programmer. Can anyone here recommend a website or forum where such a request can be made or another means to making such a request? Please let me know. Thanks.

Roger

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @TrigMan59! 

We’ve moved your post to our new Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.

**Reply by TrigMan59** [reply]
Thank you for moving this to the correct Community. I apologize for putting it into the incorrect community.

**Reply by queenofcorgis** [reply]
No need to apologize, just wanted to let you know where your post was going and share about the category 😄

**Reply by TrigMan59** [reply]
I was hesitant to post my request to this community as the Guidelines stated, "What is not allowed in this category: Asking people to code for you or homework/test questions" Thanks.
